akvo/akvo-rsr
# Issue/3205 enumerator role

**NOTE**: Based on top of #3206.  That PR can be ignored, if this is merged before that. 

  - Add an enumerator to an organisation
  - Login as the enumerator and verify that they can add/edit/delete project updates. 
  - They should also be able to post updates and generate reports (and do everything else that 'normal' users can do).
